app/preparation.py

```diff
@@ -0,0 +1,14 @@
+def get_ready(knights_config: dict) -> dict:
```

Comment:
I think it isn't the best function name :( 
Maybe smth like `set_power_and_protection` is more descriptive